test/test.js

```diff
@@ -6,7 +6,7 @@ const esbuild = require('esbuild')
 const jsonschemaPlugin = require('..')
 
 tape.test('passes schema', function (t) {
-  bundle('ok.js', null, function (err, src) {
+  bundle('ok.js', undefined, function (err, src) {
```

Comment:
This is very subjective (petpeeveesque even) but I always feel having to put a literal `undefined` somewhere is a code smell. Should these just pass `{}` instead now to make it explicit we don't want to pass options?

Alternatively, an array of arguments (with `[]` meaning no options) could be used sth like:

```js
function bundle (file, args, callback) {
  esbuild.build({
    entryPoints: [path.join(__dirname, file)],
    bundle: true,
    plugins: [jsonschemaPlugin(...args)],
    write: false
  })
    .then(function (result) {
      callback(null, result.outputFiles[0].text)
    }, function (errResult) {
      callback(errResult.errors)
    })
}
```

If this sounds too complicated / pointless for you, feel free to disregard.
```